js_modules/dagster-ui/packages/ui-core/src/search/SearchDialog.tsx

```diff
@@ -76,7 +76,7 @@ const DEBOUNCE_MSEC = 100;
 export const SearchDialog = () => {
   const history = useHistory();
   const {initialize, loading, searchPrimary, searchSecondary} = useGlobalSearch({
-    includeAssetFilters: false,
+    isCatalog: false,
```

Comment:
I renamed this to `isCatalog` since this more accurately represents what the boolean is for. It's actually only set to `true` by dagster+ though. We need to redo this composition of Plus -> OSS but considering that out of scope for now.